spectacles/cli.py

```diff
@@ -388,6 +389,13 @@ def _build_sql_subparser(
             user's branch to the revision of the branch that is on the remote. \
             WARNING: This will delete any uncommited changes in the user's workspace.",
     )
+    subparser.add_argument(
+        "--manifest-dependency",
```

Comment:
This is because of the 500 bug on the manifest endpoint. If we try to do it and they don't have a manifest file, we get a 500 and won't know if it's actually an internal server error or because they don't have a file. Therefore, for the moment, I'm suggesting that we make them have to call it if they have a manifest file.

Reply:
So I suppose the alternative is to catch the error and have a more descriptive error messaging saying something like "Unexpected error encountered via the Looker API. Please confirm you have a manifest.lkml file in your project. If you do, please submit an issue as described below." Thoughts on the tradeoffs between those approaches?

Reply:
_Most_ projects won't have a manifest.lkml file. Would you still expect to kill the run when you receive that 500?

Reply:
Ah yes, I see now, what I'm suggesting makes no sense. Thoughts on renaming the flag to `--multi-project`? Or `--import-projects`? That aligns better with what [Looker calls it](https://docs.looker.com/data-modeling/learning-lookml/importing-projects) I think. If I'm looking at the help menu for this I probably have no idea what "manifest dependency" means.

Also, this seems like something you would want to be able to set and forget in the config file or as an environment variable since it's not something you'd be changing run-to-run probably. Can you set it up to work that way as well?

Reply:
I like `--import-projects`.

Yeah, makes sense wrt the environment variables. We basically want it's actions to be both `EnvVarAction` and `"store_true"`. Is there a way to do that in our current setup? Otherwise they would need to pass `True` to the cli: `--import-projects True`.

I imagine otherwise we would possible need to write a new class of Action?